sync/azuredatabricks.py

```diff
@@ -371,11 +369,14 @@ def write_file(body: bytes):
             all_timelines = retired_timelines + list(active_timelines_by_id.values())
 
             write_file(
-                orjson.dumps(
-                    {
-                        "instances": list(all_vms_by_id.values()),
-                        "timelines": all_timelines,
-                    }
+                bytes(
+                    json.dumps(
+                        {
+                            "instances": list(all_vms_by_id.values()),
+                            "timelines": all_timelines,
+                        }
+                    ),
+                    "utf-8",
```

Comment:
I recall you saying that json dumps to a str while orjson dumps to bytes. Does the bytes format need to be adhered to here because the s3 write requires bytes? Just curious if/how this fails if you don't format it as bytes.

Reply:
Yeah, if it's not cast into bytes you get a boto3 error that says something along the lines of file obj needs to be in bytes for s3. I don't recall the exact verbiage, but it was definitely a fatal error

Reply:
Well actually, that's true for this line https://github.com/synccomputingcode/syncsparkpy/blob/f97ebc090f91dfae4a73ec1aea9133e35212e2ea/sync/awsemr.py#L755. The write_file function you're talking about here expects bytes and manipulates the body expecting it to be bytes. It throws an error: `TypeError: a bytes-like object is required, not 'str'`. It looks like dbfs expects file content to be in b64.